### PR TITLE
[AFD WAF] Add defaultState and defaultAction to managed rule set definitions

### PR DESCRIPTION
**Pull request by eschwabe**
### Overview
* Add defaultAction and defaultState to managed rule set definitions
* No API version change since only impact is to read-only API

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [X ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [X ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-03-01/webapplicationfirewall.json [hunk 1]
```diff
@@ -726,6 +718,14 @@
           "readOnly": true,
           "type": "string"
         },
+        "defaultState": {
```
Comment (lirenhe) [inline specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-03-01/webapplicationfirewall.json]: Add new property in the response is considered a breaking change.

Comment (eschwabe) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-03-01/webapplicationfirewall.json]: @lirenhe - The new fields are read-only and the associated API only supports GET requests (no PUT).

Comment (KrisBash) [reply specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-03-01/webapplicationfirewall.json]: adding read-only properties in an api version is ok from ARM guidelines

**Review by KrisBash — APPROVED** [review]
ARM sign off

**Review by lirenhe — APPROVED** [review]

**Pull request merged**